### PR TITLE
Fix /spec + super bug

### PR DESCRIPTION
**Pull request by def-**

### CODE REVIEW

**Comment by fokkonaut** [comment]
But it has to get set after the player unspecced, otherwise its still false?

**Comment by fokkonaut** [comment]
this check is also neccessary for Unsuper

**Comment by def-** [comment]
fixed, thanks.

File: src/game/server/ddracecommands.cpp [hunk 1]
```diff
@@ -123,7 +125,9 @@ void CGameContext::ConUnSuper(IConsole::IResult *pResult, void *pUserData)
 	if (pChr && pChr->m_Super)
 	{
 		pChr->m_Super = false;
-		pSelf->m_World.m_Core.m_apCharacters[pResult->m_ClientID]->m_Super = false;
+		if (pSelf->m_World.m_Core.m_apCharacters[pResult->m_ClientID]) {
```
Comment (heinrich5991) [inline src/game/server/ddracecommands.cpp:128]: Brace style.

Comment (def-) [reply src/game/server/ddracecommands.cpp:128]: Done

**Comment by Learath2** [comment]
Very interesting bug, didn't even think we removed the core from the world ever. LGTM

bors r+

**Pull request merged**